apps/cowswap-frontend/src/common/pure/TradeDetailsAccordion/index.tsx

```diff
@@ -1,28 +1,50 @@
-
 import CarretIcon from '@cowprotocol/assets/cow-swap/carret-down.svg'
 import { FiatAmount, TokenAmount } from '@cowprotocol/ui'
 import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 
 import SVG from 'react-inlinesvg'
 
+import { BridgeAccordionSummary, BridgeProtocolConfig } from 'modules/bridge'
+
 import { Wrapper, Summary, SummaryClickable, ToggleIcon, Details } from './styled'
 
+/**
+ * @property {number} [bridgeEstimatedTime] - Estimated time for bridge transaction
```

Comment:
ts-doc looks redundant here, you can add comments just to a specific parameter like you did for `bridgeEstimatedTime`